hmy/api_backend.go

```diff
@@ -360,6 +360,7 @@ func (b *APIBackend) GetMedianRawStakeSnapshot() *big.Int {
 		for i := range validator.Delegations {
 			stake.Add(stake, validator.Delegations[i].Amount)
 		}
+		stake = stake.Div(stake, big.NewInt(int64(len(validator.SlotPubKeys))))
 		stakes = append(stakes, stake)
```

Comment:
so I'm not sure of the go syntax to do it but shouldn't we have here something like : ```stakes = append (stakes, stake, stake, ..., stake)``` with the number of stake to add the number of BLS key for that validator

Reply:
True